src/store/store.ts

```diff
@@ -6,16 +6,15 @@ import getters from '@/store/getters'
 import mutations from '@/store/mutations'
 import ApplicationState from '@/types/ApplicationState'
 
-// @ts-ignore
 const { store: context } = require('@molgenis/molgenis-ui-context').default
 
 const packageJson = require('../../package.json')
 Vue.use(Vuex)
 
 context.state.appVersion = packageJson.version
 
-export default new Vuex.Store<ApplicationState>({
-  state,
+export default new Vuex.Store({
+  state: state as ApplicationState,
```

Comment:
here's where we cheat a bit, the state we're providing is not actually an ApplicationState until the context module has loaded the context